scripts/aem.js

```diff
@@ -368,7 +368,7 @@ function decorateHorizontalRules(element) {
       const hr = document.createElement('hr');
       hr.className = content === '-hr fw-' ? 'horizontal-rule horizontal-rule--full' : 'horizontal-rule';
       wrapper.appendChild(hr);
-      p.parentElement.replaceWith(wrapper);
+      p.replaceWith(wrapper);
```

Comment:
This fixes the <hr> removing its parent, which includes the documentation for the <hr>.